core/src/main/scala/kafka/metrics/KafkaMetricsReporter.scala

```diff
@@ -52,12 +52,11 @@ trait KafkaMetricsReporter {
 }
 
 object KafkaMetricsReporter {
-  val ReporterStarted: AtomicBoolean = new AtomicBoolean(false)
```

Comment:
This boolean never gets set to true, though.  Is there really any chance this is useful?